runtime/syntax/nftables.yaml

```
@@ -6,14 +6,23 @@ detect:
     
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
-    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
+    - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
     - special: "\\b(elements|hook|policy|priority|type)\\b"
     - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
     - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "[-=/:;,@]"
+    - symbol: "([-=/:;,@]|[!=])"
     - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
     - constant.string: '([\"]{1})(.*)([\"]{1})'
+    - constant.string: 
+         start: "(['\"])" 
+         end: "(['\"])" 
```

Will not work, because it identifies `"foo'` or `'bar"` as string, which they aren't. I think `''` doesn't even mark a string and thus we don't need a region with such starts/ends.
So keep it simple:
```suggestion
    - constant.string:
        start: "\""
        end: "\""
```

---

I have set it to your suggestion. It will be added with the next commit.